### DISCUSSION TITLE
Room card  entity icons with badges, and more animations

### DISCUSSION THREAD

**Discussion by CykloneFij** [andyblac/UI-Minimalist-Custom-Cards#20] [General]
Hi Andy,
Loving your custom cards, especially the room card!
Would it be possible to add badges to the entity/sensor icons? 
As well as more animations?  For example, a camera animation if recording.

Cheers

**Comment by andyblac** [comment]
badges would be too small to show any info of value, as for animations you are welcome to make your own, and submit them (for camera flashing, take a look at my Motion Sensor animation, you should be Abel to use it as a base). I would be happy to add via a Pull Request.

**Reply by CykloneFij** [reply]
Yeah, but more as a flag for a state on the icons.
For example, an entity on the right side.  I use frigate cameras and currently use colours to show if it's in detect state as not all my cameras are recording, they are mostly for viewing.  The ones on detect have a notify automation that isn't always on, so a flag/badge to indicate that state.  The badge size can be set in css yes?  For badges that may show some counter or something?
I will look into the animations you have and look into making one.
Cheers
